client/src/components/App/Content/InvestigationForm/TabManagement/TabManagement.tsx

```diff
@@ -89,4 +89,5 @@ export default TabManagement;
 interface Props {
     currentTab: TabObj;
     setCurrentTab: (currentTab: TabObj) => void;
+    onTabClicked: () => void;
```

Comment:
Why didn't you save for the rest of the tabs?

Reply:
יוצאים מנקודת הנחה שזה כבר נשמר במעבר בין הטאבים ורק צריך לשמור את הטאב הנוכחי עליו אנחנו נמצאים

Reply:
פשוט יש שם רק 2 פןנקציות של שמירה עבור 2 טאבים ולא עבור כולם

Reply:
ההערה הייתה סתם שם.. אבל יש שם 2 פונקציות חסרות, כנסי לקוד